utils/src/test/kotlin/ArchiveUtilsTest.kt

```diff
@@ -76,5 +77,43 @@ class ArchiveUtilsTest : StringSpec() {
             fileB.exists() shouldBe true
             fileB.readText() shouldBe "b\n"
         }
+
+        "Debian deb archive can be unpacked" {
+            val archive = File("src/test/assets/testpkg.deb")
+
+            archive.unpack(outputDir)
+
+            val extractedScriptFile = outputDir.resolve("usr/bin/test")
+            extractedScriptFile.exists() shouldBe true
+
+            val extractedControlFile = outputDir.resolve("control")
+            extractedControlFile.exists() shouldBe true
+            val expectedControl = File("src/test/assets/control-expected.txt").readText()
+            extractedControlFile.readText() shouldBe expectedControl
+
+            listOf("data.tar.xz", "control.tar.xz").forEach { tarFileName ->
```

Comment:
Should we use `DEBIAN_ARCHIVE_CONTENT` here? Or should we exactly *not* to that for testing?

Reply:
I opted for the latter. Also, the constant is private.

Reply:
> Also, the constant is private.

Well, you could easily make it `internal` (which would be my personal preference).